composer.json

```diff
@@ -67,7 +67,7 @@
         "gregwar/captcha-bundle": "^2.0",
         "incenteev/composer-parameter-handler": "^2.1.3",
         "jms/payment-core-bundle": "~1.3",
-        "kaliop/ezmigrationbundle": "^4.7",
+        "kaliop/ezmigrationbundle": "^5",
```

Comment:
We don't use single digit versions. This should be ^5.0

Reply:
@mauroaltamura could you please address this remark?